effekt/shared/src/main/scala/effekt/machine/PrettyPrinter.scala

```diff
@@ -83,6 +84,8 @@ object PrettyPrinter extends ParenPrettyPrinter {
 
     case LiteralDouble(name, value, rest) =>
       "let" <+> name <+> "=" <+> value.toString <> ";" <> line <> toDoc(rest)
+
+    case LiteralUTF8String(name, utf8, rest) => ???
```

Comment:
Can't we show it here?

Reply:
We surely can; for what is `machine/PrettyPrinter.scala` used? What format would be appropriate?

Reply:
Ref. [f442b59](https://github.com/effekt-lang/effekt/pull/142/commits/f442b59c9c85ad630d36489fea752326b5e73a5f)